mapbox/validation.py

```diff
@@ -0,0 +1,31 @@
+
+class MapboxValidationError(ValueError):
+    pass
+
+
+class InvalidPlaceTypeError(MapboxValidationError):
```

Comment:
The dict is more than just a validation detail for geocoding. It's part of our SDK: https://github.com/mapbox/mapbox-sdk-py/blob/master/mapbox/services/geocoding.py#L82. I'd like to do this for distance and directions too, because it's a handy feature.